isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
What is `read_pos_after` if the variant is at the end of the read? 

Should there be a separate check for deletions that the `ref` matches something?